qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +367,55 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def particle_number(n_orbitals, mapping="jordan_wigner"):
+    r"""Computes the particle number operator :math:`\hat{N}=\sum_\alpha \hat{n}_\alpha` in
+    the Pauli basis. Its eigenvalue is the number of particles in a given state of the
+    many-body system.
+
+    The particle number operator reads,
+
+    .. math::
+
+        \hat{N} = \sum_\alpha \hat{c}_\alpha^\dagger \hat{c}_\alpha.
+
+    The index :math:`\alpha` runs over the basis of single-particle orbitals
+    :math:`\vert \alpha \rangle` and :math:`\hat{c}^\dagger` and :math:`\hat{c}` are
+    the particle creation and annihilation operator, respectively.
```

Comment:
```suggestion
    :math:`\vert \alpha \rangle`. The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}` are
    the particle creation and annihilation operators, respectively.
```
To avoid repeated use of "and"